### ISSUE TITLE
training set score improving while validation set score without change at all

### ISSUE THREAD

**Issue by thebeancounter** [adambielski/siamese-triplet#22]
Hello 

Does it make any sense that my training performance improve (different data than mnist) while the performance over the validation set don't change -at all-, I checked, both of the sets are balanced in labels, and include the same types of labels.

**Comment by thebeancounter** [comment]
found that this can be referred to the triplet selection strategies

**Comment by Crysflair** [comment]
> found that this can be referred to the triplet selection strategies

Hi, can you elaborate a little on this? Thank you.
